src/importlinter/domain/helpers.py

```diff
@@ -121,6 +121,29 @@ def resolve_import_expressions(
     return (resolved_imports, unresolved_expressions)
 
 
+def resolve_module_expressions(
```

Comment:
Good call on the move to `domain.helpers`, it makes more sense to have this alongside `import_expression_to_imports`.

Couple of things.

1. Maybe it would be more consistent to name this `module_expressions_to_modules`, considering the naming of `import_expression_to_imports`. What do you think?
2. Would it be better to return a `set[Module]` here? Having it as an iterable implies that both ordering and duplicates are possible.

Reply:
Yeah, good idea, I'll rename it.